cloudflare_rules/cloudflare_firewall_high_volume_events.py

```diff
@@ -0,0 +1,18 @@
+from panther_cloudflare_helpers import map_source_to_name
+
+
+def rule(event):
+    if event.get("Action") != "block":
```

Comment:
Mostly just curious on why you inverted this instead of action == block?

Reply:
If the http requests were blocked, then that wouldn't have any impact on live services. If there was a large number of events blocked, then that would indicate some type of "anomalous" events that largely had no impact

Reply:
If the http requests were blocked, then that wouldn't have any impact on live services. If there was a large number of events blocked, then that would indicate some type of "anomalous" events that largely had no impact